crates/primitives/src/attr.rs

```diff
@@ -1,5 +1,59 @@
-use bitflags::bitflags;
+//! Constraints and Column attributes.
+//!
+//! ## For tables
+//!
+//! The database engine support a sub-set of constraints defined by the `SQL` standard:
+//!
+//! ### UNIQUE:
+//!
+//!  One or many columns with enforced uniqueness using an auto-generate index.
+//!
+//! ### IDENTITY:
+//!
+//! A column that is made up of values generated by the database using a sequence.
+//!
+//!  Differs from `PRIMARY_KEY` in that its values are managed by the database and usually cannot be modified.
```

Comment:
NOTE: Currently `IDENTITY` implies `UNIQUE` but that is wrong: https://en.wikipedia.org/wiki/Identity_column.

However, to change this need we review how others are using this.